platform-db-runtime/src/main/java/com/softicar/platform/db/runtime/table/listener/DbTableRowNotificationSet.java

```diff
@@ -1,19 +1,19 @@
 package com.softicar.platform.db.runtime.table.listener;
 
+import com.softicar.platform.common.container.list.HashList;
 import com.softicar.platform.db.runtime.table.IDbTable;
 import java.util.Collection;
-import java.util.HashSet;
-import java.util.Set;
+import java.util.List;
 
 class DbTableRowNotificationSet<R> implements IDbTableRowNotificationSet<R> {
 
 	private final IDbTable<R, ?> table;
-	private final Set<DbTableRowNotification<R>> notifications;
+	private final List<DbTableRowNotification<R>> notifications;
 
 	public DbTableRowNotificationSet(IDbTable<R, ?> table) {
 
 		this.table = table;
-		this.notifications = new HashSet<>();
+		this.notifications = new HashList<>();
```

Comment:
the set was preventing a duplication, if we change it to `HashList`, this is no longer the case

Reply:
Yes it is. That's the whole point of `HashList`. See `HashList.add`:
```
	@Override
	public boolean add(T element) {

		if (set.add(element)) {
			list.add(element);
			return true;
		} else {
			return false;
		}
	}
```

Reply:
you are right, I forgot about that :)